docs/topics/api-guidelines-build-for-multiplatform.md

```diff
@@ -0,0 +1,84 @@
+[//]: # (title: Building a Kotlin library for multiplatform)
+
+When creating a Kotlin library, consider building and [publishing the library with support for Kotlin Multiplatform](multiplatform-publish-lib.md).
+This broadens the target audience of your library, making it compatible with projects targeting multiple platforms.
+
+The following sections provide guidelines to help you build a Kotlin Multiplatform library effectively.
+
+## Maximize your reach
+
+To make your library available to the largest number of projects as a dependency,
+aim to support as many [target platforms](multiplatform-dsl-reference.md#targets) of Kotlin Multiplatform as possible.
+If your library doesn’t support the platforms used by a multiplatform project,
+whether it’s a library or an application, it becomes difficult for that project to depend on your library.
+Projects might use your library for some platforms and implement separate solutions for others,
```

Comment:
That's a nice way to emphasize - the only thing I think might work better is to keep the phrasing here: 
"...they will choose an alternative library altogether that supports all their platforms." 
This way I think it focuses the part that they will just choose another library instead of supporting all the platforms

Reply:
Yeah, I agree